core/shared/src/main/scala/scalaz/zio/RefM.scala

```diff
@@ -34,7 +34,7 @@ import scalaz.zio.Exit.Cause
  * } yield ()
  * }}}
  */
-final class RefM[A] private (value: Ref[A], queue: Queue[RefM.Bundle[A, _]]) extends Serializable {
+final class RefM[E, A] private (value: Ref[A], queue: Queue[RefM.Bundle[E, A, _]]) extends Serializable {
```

Comment:
Instead of introducing the type parameter here, consider introducing it below, on the relevant methods.